### DISCUSSION TITLE
User indicator icons size in a vertical budgie panel

### DISCUSSION THREAD

**Discussion by hakimjonas** [BuddiesOfBudgie/budgie-desktop#463] [Q&A] [answered]
Edit: Using UserIndicator as an example since this is not limited to this applet.

Using budgie in a fully updated Solus system, every icon in my panel (left panel - not in dock mode) seem to ocupy a square if you include the surrounding space. I belive this is good and well. I have chosen to also include the user indicator applet. But it has a small icon and also the surrounding space is not a square like the other icons. It gets the full width of the panel but the height is only a few pixels more than the icon, but making it stand out visually on mouseover but also makes it harder to click. 

I belive at least the issue of the surrounding "box" for lack of a better word should be fixed and atleast for the usability aspect. Wether the icon should also be scaled up for this usecase I guess can be discussed.

**Comment by JoshStrobl** [comment, marked as answer]
This should be a bug report not a discussion. Please choose Bug report from https://github.com/BuddiesOfBudgie/budgie-desktop/issues/new/choose :)
